app/components/primer/beta/flash.rb

```diff
@@ -11,18 +11,16 @@ class Flash < Primer::Component
       # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
       renders_one :action, lambda { |**system_arguments|
         deny_tag_argument(**system_arguments)
-        system_arguments[:tag] = :div
-        system_arguments[:classes] = class_names(system_arguments[:classes], "flash-action")
 
-        Primer::BaseComponent.new(**system_arguments)
+        Primer::Beta::Button.new(**system_arguments)
```

Comment:
Apologies if still working on this, are we adding back support for `forms, include-fragment, details` here?

Reply:
Yeah sorry, I haven't gotten a chance to switch back to `BaseComponent` yet. Actually thinking about making this a polymorphic slot.